force-app/main/default/classes/Homework.cls

```diff
@@ -18,31 +18,63 @@ public with sharing class Homework
     // 1. Take a List of integers and return the largest value using a loop
     public Integer maxInListLoop(List<Integer> values)
     {
+        Integer max = values[0];
```

Comment:
Good initialisation - outside of the contents of the collection, there is no sensible default.